subprojects/distributions-dependencies/build.gradle.kts

```
@@ -58,7 +58,7 @@ dependencies {
         api(libs.googleHttpClient)      { version { strictly(googleApiVersion); because("our GCS version requires 1.25.0") }}
         api(libs.googleHttpClientJackson2) { version { strictly(googleApiVersion); because("our GCS version requires 1.25.0") }}
         api(libs.googleOauthClient)     { version { strictly(googleApiVersion); because("our GCS version requires 1.25.0") }}
-        api(libs.gradleProfiler)        { version { strictly("0.17.0-alpha07") }}
+        api(libs.gradleProfiler)        { version { strictly("0.17.0-alpha09") }}
```

```suggestion
        api(libs.gradleProfiler)        { version { strictly("0.17.0-alpha10") }}
```

The new version is already out.